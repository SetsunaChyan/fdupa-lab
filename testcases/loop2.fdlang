function main(){
x = input();
y = input();

while (y >= 1) {
    check_interval(y, 1, 10);
    if (x <= 10) {
        y = y + 1;
    } else {
        y = y - 1;
    }
}

check_interval(y, 0, 11);
}
