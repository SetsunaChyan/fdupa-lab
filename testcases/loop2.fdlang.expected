Line 6:  NO; y in [1, 255]
Line 14: YES; y in [0, 0]
